### PR TITLE
Update data explorer indexer to run on Python 3

### PR DESCRIPTION
**Pull request by wnojopra**
Tested by indexing amp_pd, 1000_genomes, and e2e tests.

### CODE REVIEW

File: bigquery/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # When running this Dockerfile context must be project root, in order to pick
 # up indexer_util and dataset_config
 
-FROM python:2
+FROM python:3
```
Comment (melissachang) [inline bigquery/Dockerfile]: Please make branch in this repo, so circleci will run.

Comment (melissachang) [inline bigquery/Dockerfile]: Please also test that this works on GCP, in your test project(s).

- 1000 Genomes
- Framingham
- The new AMP PD data
- NHS
- baseline_second_data_release

If you could leave the clusters around so I can double-check, that would be great.

Comment (wnojopra) [reply bigquery/Dockerfile]: I will create a new PR from a branch in this repo.

**Comment by wnojopra** [comment]
See https://github.com/DataBiosphere/data-explorer-indexers/pull/152 instead

**Pull request closed without merging**